.github/workflows/commit-message-check.yml

```diff
@@ -0,0 +1,44 @@
+name: Check Commit Message
+
+on: [push]
+
+jobs:
+  check-commit-message:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Get pushed commits
+        id: get_commits
+        run: echo "::set-output name=commits::$(git log ${{ github.event.before }}..${{ github.event.after }} --pretty=format:'%s')"
```

Comment:
This line causes a warning: https://github.com/butvinm/nobutt/pull/15/checks#step:3:6

Reply:
Uh oh, sure